config/nginx.conf

```diff
@@ -0,0 +1,26 @@
+upstream gb-botler {
+  server unix:///var/www/gb-botler/shared/sockets/puma.sock;
+}
+
+server {
+  listen 80;
+  
+  root /var/www/gb-botler/current/public;
+
+  try_files $uri/index.html $uri @gb-botler;
+
+  access_log /var/www/gb-botler/shared/log/nginx.out;
+  error_log /var/www/gb-botler/shared/log/nginx.err;
+
+  location ~ ^/(system|uploads|assets)/ {
+    expires max;
+  }
+
+  location @gb-botler {
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    proxy_set_header X-Forwarded-Proto $scheme;
+    proxy_set_header Host $http_host;
```

Comment:
@naps62 Finally found the solution. I had to set this to `$http_host`, which is the value of the Host header in the client's request.